source/includes/windows/common-minio-kes.rst

```diff
@@ -141,6 +142,12 @@ MinIO uses the :envvar:`MINIO_KMS_KES_KEY_NAME` key for the following cryptograp
   include a specific |EK|.
 - Encrypting objects using :ref:`SSE-S3 <minio-encryption-sse-s3>`.
 
+MinIO uses the :envvar:`MINIO_KMS_KES_ENCLAVE` key to define the name of the KES enclave to use.
```

Comment:
```suggestion
MinIO uses the :envvar:`MINIO_KMS_KES_ENCLAVE` key to define the name of the KES enclave to use. Each enclave isolates it's associated keys from any other on the KES server.
```

Suggest making this change throughout. That way users do not have to make the jump to understand what an enclave is.

Reply:
Added an explanatory note in each of the platform files.